### PR TITLE
Rewrite .stock message

### PR DESCRIPTION
**Pull request by PixelSergey**
- Rewrote the message in the stock command to incorporate The Guide:tm: for the frogminer method.
- Cleared up some messy grammar
- Removed rarely - if ever - used aliases (like .stock114, .stock115, etc. This can be reverted if you think these aliases were a vital feature).

### CODE REVIEW

**Comment by FrozenChen** [comment]
Due to recent changes, the conflicts must be resolved.
Also the assistance cog now uses multiline strings for long strings for better visibility.

File: addons/assistance.py [hunk 1]
```diff
@@ -243,24 +243,23 @@ def check_console(self, message, channel, consoles):
...
-                - [NTRBoot](https://3ds.hacks.guide/ntrboot) which requires a compatible NDS flashcart and maybe an\
-additional DS(i) or hacked 3DS console depending on the flashcart
-                - [Frogminer](https://3ds.filthycasuals.tech/frogminer.html) which requires a homebrew entrypoint like \
-[steelminer](http://steelminer.jisagi.net/) (free method) or freakyhax.
-                - [Seedminer](https://3ds.hacks.guide/installing-boot9strap-\(seedminer\)) which requires a compatible \
-DSiWare game
-                - [Hardmod](https://3ds.hacks.guide/installing-boot9strap-\(hardmod\)) which requires "soldering \
-**Not for beginners!**
-                **Downgrading is impossible on 11.4+!**
+                You have many possible options for installing CFW:
```
Comment (FrozenChen) [inline addons/assistance.py]: i
https://www.python.org/dev/peps/pep-0008/#maximum-line-length
The change to multilines was for this very reason. At most 120 chars per line

Comment (PixelSergey) [reply addons/assistance.py]: Ya got me

File: addons/assistance.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def check_console(self, message, channel, consoles):
             await self.bot.say(msg)
             return
         await self.bot.delete_message(ctx.message)
-        # await self.bot.say("Request sent.")
```
Comment (FrozenChen) [inline addons/assistance.py]: Why was this line changed?

**Comment by noirscape** [comment]
I personally think that the final link to the guide is a little bit pointless as `.guide 3ds` is a thing.

I think using embed fields could be much more useful for these methods (title of field -> method, description -> short description and link, markdown can be used in the description of the field). They'd have to be set to `inline=false` though, or else it'll just get messy.

I think that adding output to sr should still remain a thing or it should at least be timed(?) but this seems beyond the OG PR.

A potential (unrelated) suggestion would be to restrict this command from being used (or expanded to be suitable to a la `.guide`) in the switch and wiiu assistance channels, as the command name is rather generic.

Just my 0.02, but I ain't staff.

**Comment by FrozenChen** [comment]
> I think that adding output to sr should still remain a thing or it should at least be timed(?) but this seems beyond the OG PR.

@noirscape That thing has been there commented for ages, sergey just removed `say` for some reason probably a mistake

**Comment by noirscape** [comment]
@FrozenChen ah, okay. The diff kinda confused that.

**Comment by FrozenChen** [comment]
Changes werent made in half a month
